### PR TITLE
Pin opm to 2021.04

### PR DESCRIPTION
**Pull request by asnyv**

### CODE REVIEW

**Comment by asnyv** [comment]
Temporary pin to pass tests and enable a release of `v0.16.0`

File: setup.py [hunk 1]
```diff
@@ -18,7 +18,7 @@
 REQUIREMENTS = [
     "ecl",
     "numpy",
-    "opm>=2020.10.2",  # NB: Pypi versions.
+    "opm==2021.04",  # NB: Pypi versions.
```
Comment (berland) [inline setup.py:21]: Looking at https://pypi.org/project/opm/#history, I would guess 2021.04.0 is the correct? Or it might find it still?

**Comment by asnyv** [comment]
Hmm, yes seems like you are right. Quite messy version numbering in `opm` it seems. `2021.04` worked though, but I can of course change it 😅
image

File: setup.py (hunk 1 above)
Comment (asnyv) [reply setup.py:21]: Hmm, yes seems like you are right. Quite messy version numbering in `opm` it seems. `2021.04` worked though, but I can of course change it 😅
image

**Review by berland — APPROVED** [review]
LGTM!

**Pull request merged**